internal/utils/validation.go

```diff
@@ -53,3 +54,11 @@ func Validate(s interface{}) error {
 
 	return fmt.Errorf("%s has %d errors:\n%s", ty, len(err.(validation.ValidationErrors)), msg)
 }
+
+func ValidateCertificate(cert string) error {
+	certPool := x509.NewCertPool()
+	if !certPool.AppendCertsFromPEM([]byte(cert)) {
```

Comment:
That's not really what this function is supposed to check. 

> // AppendCertsFromPEM attempts to parse a series of PEM encoded certificates.
// It appends any certificates found to s and reports whether any certificates
// were successfully parsed.

If there is a single valid certificate inside the bytestring and then a bunch of garbage, this succeeds

Also a nit: the return value is never used in its form and could be a bool with as `IsValidCert` or similar
```suggestion
	return certPool.AppendCertsFromPEM([]byte(cert))
```